### PR TITLE
Add option to not list add-on (bug 1121236)

### PR DESCRIPTION
**Pull request by magopian**
Fixes [bug 1121236](https://bugzilla.mozilla.org/show_bug.cgi?id=1121236)

### CODE REVIEW

File: migrations/805-add-addon-is_listed.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE addons
```
Comment (clouserw) [inline migrations/805-add-addon-is_listed.sql]: Do we know how long this takes to run?

**Comment by clouserw** [comment]
r+

File: migrations/805-add-addon-is_listed.sql (hunk 1 above)
Comment (magopian) [inline migrations/805-add-addon-is_listed.sql]: @jasonthomas? From what I can remember, adding a column with a default to the addons table takes a few minutes, could you please confirm?

Comment (jasonthomas) [inline migrations/805-add-addon-is_listed.sql]: I think in the past a ALTER on the addons table will take about 10-15 minutes. We can test on dev and stage to have a better idea how long it takes.

Comment (jasonthomas) [inline migrations/805-add-addon-is_listed.sql]:
```
Running SQL migration 805:
ALTER TABLE addons
    ADD COLUMN `is_listed` bool NOT NULL DEFAULT true;

That took 57.80 seconds
```

**Pull request merged**